components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.groups/public/js/listing.js

```diff
@@ -198,6 +208,11 @@ function loadGroups() {
                     $(this).attr('data-search', data.owner);
                     $(this).attr('data-display', data.owner);
                     break;
+                case 3:
```

Comment:
It's good to give proper naming on case scenarios, otherwise its hard to understand the logic